apps/vue-showcase/src/app/router.ts

```diff
@@ -31,12 +31,52 @@ const router = createRouter({
       component: () => import('./views/HomeView.vue'),
       meta: { title: 'vue-showcase', requiresAuth: false },
     },
+    {
+      path: '/route-guard',
+      name: 'RouteGuard',
+      component: () => import('./views/RouteGuardView.vue'),
+      meta: { title: 'vue-route-guard-view', requiresAuth: false },
+      children: [
+        {
+          path: '/route-guard',
+          name: 'RouteGuardHome',
+          component: () => import('./views/RouteGuardHomeView.vue'),
+          meta: { title: 'vue-route-guard-home-view', requiresAuth: true },
+        },
+        {
+          path: '/route-guard/login',
+          name: 'RouteGuardLogin',
+          component: () => import('./views/RouteGuardLoginView.vue'),
+          meta: { title: 'vue-route-guard-login-view', requiresAuth: false },
+        },
+        {
+          path: '/route-guard/about',
+          name: 'RouteGuardAbout',
+          component: () => import('./views/RouteGuardAboutView.vue'),
```

Comment:
For the views, how about instead of naming them `RouteGuardx`, you create directories to group such things. For example:
- `views/errors/` for errors
- `views/auth/` for those that require you to be logged out to view
- `views/logged-in`  for those that require an individual to be logged in

I think this would be clearer than having to follow a convention that may or may not be implicit. It is easy for someone to forget to add `RouteGuard*` to their view name but having it in the `/logged-in` directory seems a little more obvious. Naming could be flexible but I think the general idea could help push further in the right direction.